From a79b948cd9235d90c4bf3bd356d814f2b61ac7c0 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Tue, 10 Nov 2015 13:35:51 +0200
Subject: [PATCH 0485/1240] fix: env: added missing environment variable

- Added missing environment variable:
  serverip, ipaddr, ramfs_addr

Change-Id: I8c3c449493f71d6ad149d0affdbb38dc1d08fd43
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24706
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 include/configs/mvebu-common.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index b9a87c5..1038a5b 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -128,6 +128,7 @@
 					"loadaddr=0x2000000\0"		\
 					"fdt_high=0xffffffffffffffff\0"	\
 					"hostname=marvell\0"		\
+					"ramfs_addr=0x3000000\0"	\
 					"ramfs_name=-\0"		\
 					"fdt_name=fdt.dtb\0"		\
 					"netdev=eth0\0"			\
@@ -329,8 +330,8 @@
 /* Add network parameters when network command is enabled */
 #ifdef CONFIG_CMD_NET
 	/* Environment */
-	#define CONFIG_IPADDR			/* In order to cause an error if not set */
-	#define CONFIG_SERVERIP			/* In order to cause an error if not set */
+	#define CONFIG_IPADDR		0.0.0.0	/* In order to cause an error */
+	#define CONFIG_SERVERIP		0.0.0.0	/* In order to cause an error */
 	#define CONFIG_NETMASK          255.255.255.0
 	#define CONFIG_GATEWAYIP	10.4.50.254
 	#define CONFIG_ETHADDR          00:00:00:00:51:81
-- 
1.9.1

